### DISCUSSION TITLE
MountainDuck: I get notifications for an disabled notification

### DISCUSSION THREAD

**Discussion by muescha** [iterate-ch/cyberduck#15041] [Q&A] [answered]
**Describe the bug**

I got from time to time the notification about a "disconnected" ftp connection

**To Reproduce**

just wait and get the error from time to time.

**Expected behavior**

I disabled the notification, so I expect no notification.

**Screenshots**
Settings:

<img width="681" alt="Bildschirmfoto 2023-08-27 um 18 36 55" src="https://github.com/iterate-ch/cyberduck/assets/184316/14e9ad55-b2cf-4a04-ae26-eb65d5814c72">

Error Message:
<img width="322" alt="Bildschirmfoto 2023-08-28 um 12 24 29" src="https://github.com/iterate-ch/cyberduck/assets/184316/c33e7feb-912f-4dd5-a6c2-2ba15a58e147">

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.4 (22F66)
 - Version: MountainDuck Version 4.14.3 (21432)

**Log Files**

the errors with the same time stamps are always similar (changed domain to `domain.com` and ip to `1.2.3.4`)

```log
2023-08-27 18:31:01,082 [sshj-Reader-ftp.domain.com/1.2.3.4:22-1693149495884] ERROR net.schmizz.sshj.transport.TransportImpl - Dying because - Operation timed out
java.net.SocketException: Operation timed out
	at sun.nio.ch.NioSocketImpl.implRead(Unknown Source) ~[?:?]
	at sun.nio.ch.NioSocketImpl.read(Unknown Source) ~[?:?]
	at sun.nio.ch.NioSocketImpl$1.read(Unknown Source) ~[?:?]
	at java.net.Socket$SocketInputStream.read(Unknown Source) ~[?:?]
	at net.schmizz.sshj.transport.Reader.run(Reader.java:50) [sshj-0.35.0.jar:?]
2023-08-27 18:31:01,088 [NFS Worker Thread(3)] ERROR net.schmizz.concurrent.Promise - <<sftp / 8>> woke to: net.schmizz.sshj.sftp.SFTPException: Operation timed out
```

**Additional context**

I have 2 ftp connections, but randomly only one of them get the error and is shown in the dialogbox.

**Comment by dkocher** [comment, marked as answer]
The alert from Finder.app about the mounted volume being unresponsive is independent from the notifications sent to the notification center of macOS. We are aware of the alert appearing in some contexts and are looking into possible remedy to suppress it. Usually it will disappear automatically when you access the volume in Finder.app next time.

**Reply by muescha** [reply]
Since I not always access volume, I am not interested to see a disconnecting information when it is just in the background.

To understand this message: it comes from finder and not from mountain duck? Or is it the result of the time out of mountain duck?

**Reply by tatze96** [reply]
Correct, the message comes from Finder, not from Mountain Duck. The notification usually occurs when Finder closes the connection because Mountain Duck hasn't answered fast enough on the request of Finder. Although Finder indicates that the connection is interrupted, it isn't, as Mountain Duck is still connected to the server.

This is an issue within the operating system that can occur to any network drive.

The mounted drive will stay connected if you click on "Ignore" to dismiss the notification or wait until it disappears by itself.

**Reply by dkocher** [reply]
We had another fix in version _[4.17.0](https://mountainduck.io/changelog/)_ that should ensure the _Server connections interrupted_ prompt is not shown in Finder. Let us know if you can still reproduce and if there is a pattern.

**Reply by muescha** [reply]
> Let us know if you can still reproduce and if there is a pattern.

Thx. 

I will monitor this to see if it happens again.
